src/compositions/Video/Video.js

```diff
@@ -31,6 +27,7 @@ const Video = ({ video }) => {
           onStateChange={ onStateChange }
         />
       </div>
+      <h4>You must watch the full video without stopping before taking the survey.</h4>
```

Comment:
Will this text change depending on the language?

Reply:
not yet needs to though